src/Package/Impl/Help/HelpVisualComponent.cs

```diff
@@ -156,7 +206,11 @@ internal sealed class HelpVisualComponent : IHelpVisualComponent {
         }
 
         private void OnNavigating(object sender, WebBrowserNavigatingEventArgs e) {
-            if (Browser.Document != null && Browser.Document.Window != null) {
+            // Disconnect browser from the tool window so it does not
+            // flicker when we change page and element styling.
+            _host.Child = null;
```

Comment:
Probably not. If we created Browser, we created host and we never set it to null